SECURITY.md

```diff
@@ -1,6 +1,10 @@
 # Security Policy
 
-Full details of the WordPress Security Policy can be found on [HackerOne](https://hackerone.com/wordpress). You can also read more in a detailed white paper about [WordPress Security](https://wordpress.org/about/security/).
+WordPress is an open-source publishing platform. The WordPress Security Team believes in Responsible Disclosure by alerting the security team immediately and privately of any potential vulnerabilities.
+
+Our HackerOne program covers the Core software, as well as a variety of related projects and infrastructure.
+
+Full details of the WordPress Security Policy and the list of covered projects and infrastructure can be found on [HackerOne](https://hackerone.com/wordpress). You can also read more in a detailed white paper about [WordPress Security](https://wordpress.org/about/security/).
 
 ## Supported Versions
```

Comment:
It's outside the diff but let's remove the line that follows, it's from the template used to set up this file.

```diff
- Use this section to tell people about which versions of your project are
- currently being supported with security updates.